config.properties

```diff
@@ -0,0 +1,27 @@
+[manifestPropertyNames]
+org.eclipse.xtext=xtextVersion
+org.eclipse.xtend=xtextVersion
+org.eclipse.lsp4j=lsp4jVersion
+org.opentest4j=openTest4jVersion
+org.eclipse.core.runtime=runtimeVersion
+org.eclipse.core.resources=resourcesVersion
+org.junit.jupiter=jupiterVersion
+
+[versions]
+commonsCliVersion=1.4
+jacocoVersion=0.8.7
+juiceVersion=5.0.1
+jupiterVersion=5.8.2
+jUnitPlatformVersion=1.8.2
+jUnitVersion=4.13.2
+kotlinJvmTarget=11
+kotlinVersion=1.6.10
```

Comment:
I think the current compile errors will go away when Maven/P2 deps also use Kotlin `v1.6.10`. My guess is that they currently don't?

Reply:
turns out the kotlin versions in `pom.xml` didnt get updated yet. I ran the script to update those and now the build test is passing